core/backend/src/rpc/tracing.ts

```diff
@@ -64,10 +65,8 @@ export function initializeTracing(enableOpenTelemetry: boolean = false) {
 
   if (enableOpenTelemetry) {
     try {
-      // eslint-disable-next-line @typescript-eslint/no-require-imports
-      const api = require("@opentelemetry/api");
```

Comment:
i believe the way this was done before (require in a try block) was to allow for this pkg to become an optional peer; and the changes you made make it a depenendency now which im not sure we want

cc @paulius-valiunas